learn-stan/stancon-talks.qmd

```diff
@@ -2,6 +2,28 @@
 title: StanCon Talks
 page-layout: full
 toc-location: right
+
+aliases:
+ - ../events/stancon2019Cambridge/index.html
+ - ../events/stancon2019Cambridge/abstracts.html
```

Comment:
The links for a specific year could link to the header, e.g. both of these could be in redirects.txt with ` https://mc-stan.org/learn-stan/stancon-talks.html#stancon-2019-cambridge`